### PR TITLE
Code refactor for spacing classes!

### PR DESCRIPTION
**Pull request by IamManchanda**
This **excludes** #10568

Just implemented DRY coding a little bit!

### CODE REVIEW

File: scss/prototype/_spacing.scss [hunk 1]
```diff
@@ -66,135 +66,100 @@ $prototype-spacers-count: 3 !default;
...
-    }
-
-    .padding-bottom-#{$spacer} {
-      @include padding(null, null, $spacer, null);
-    }
+    
+    @each $prop in (margin, padding) {
+      // All Sides
+      .#{$prop}-#{$spacer} {
+        @if ($prop == margin) { @include margin($spacer, $spacer, $spacer, $spacer); }
```
Comment (kball) [inline scss/prototype/_spacing.scss]: I like the loops, but lets have the `@include` on its own line, so more like
```
@if ($prop == margin) {
  @include margin($spacer, $spacer, $spacer, $spacer);
} @else if ($prop == padding) {
  @include padding($spacer, $spacer, $spacer, $spacer);
}

**Comment by IamManchanda** [comment]
done @kball Indentation and also some more cool refactor 

Also,

This removes the dirt of 

```
margin-top: 1rem;
margin-right: 1rem;
margin-bottom: 1rem;
margin-left: 1rem;
```

**Comment by IamManchanda** [comment]
@kball As discussed, I have removed the map for now, as we would have to then migrate it!

I am closing that above linked PR and will push a new for same (6.5) after this PR merges!

**Comment by IamManchanda** [comment]
Any updates @kball

**Pull request merged**